### ISSUE TITLE
请问可以支持Texture这个库做骨架屏吗?

### ISSUE THREAD

**Issue by bianzhifeng** [tigerAndBull/TABAnimated#156]
### New Issue Checklist

请问可以支持Texture这个库做骨架屏吗?

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. iOS / macOS
 Platform Version        | e.g. 12.0 / 10.14.0
 TABAnimated Version      | e.g. 5.0.0 / 4.4.0
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by tigerAndBull** [comment]
暂无做新计划
